docs/Repository/README.md

```diff
@@ -0,0 +1,46 @@
+## Garden Linux Repositories
+
+Garden Linux is a Debian-based Operating System. Therefore it uses apt.
+
+The location of sources are at `/etc/apt/sources.list` respectively in the folder `/etc/apt/sources.list.d/`
+
+#### Default Repo:
+	deb http://repo.gardenlinux.io/gardenlinux dev main
```

Comment:
In a sense you are right, `dev` is the default repo if you want to have always the latest packages. 

However, for long term releases this is a little bit more complicated. The `dev` repo can contain major package updates, but a long term release does not automatically update major version. 

An example is openssl1.1.1 vs openssl3, and the respective dependency to libc versions. 


long story short; maybe we should not use the word "default" here.